gns3server/compute/docker/__init__.py

```diff
@@ -54,6 +58,25 @@ def __init__(self):
         self._session = None
         self._api_version = DOCKER_MINIMUM_API_VERSION
 
+    @staticmethod
+    def install_busybox():
+
+        if not sys.platform.startswith("linux"):
+            return
+        dst_busybox = os.path.join(os.path.dirname(os.path.abspath(__file__)), "resources", "bin", "busybox")
+        if os.path.isfile(dst_busybox):
+            return
+        for busybox_exec in ("busybox-static", "busybox.static", "busybox"):
+            busybox_path = shutil.which(busybox_exec)
+            if busybox_path:
+                log.info(f"Installing busybox from '{busybox_path}'")
+                try:
+                    shutil.copy2(busybox_path, dst_busybox, follow_symlinks=True)
+                    return
+                except OSError as e:
+                    raise DockerError(f"Could not install busybox: {e}")
```

Comment:
@b-ehlers

I have migrated the code found in setup.py however why did you need to call the `ldd` command?

https://github.com/GNS3/gns3-server/blob/3.0/setup.py#L34L39

Thanks.

Reply:
Oh yes I remember now. Thanks for reminding me.